cloud/README.md

```diff
@@ -0,0 +1,42 @@
+## How to get OAuth2 options
+
+When configuring and using OAuth2, we need to specify the following parameters:
```

Comment:
change "we" into "you". Check all similar cases.
Use second person, for details, refer to https://developers.google.com/style/person